### PR TITLE
[Notifier] Add links to push bridges' README

### PR DESCRIPTION
**Pull request by mikaelkael**
Following https://github.com/symfony/symfony-docs/pull/18254 and fix merge issue added by https://github.com/symfony/symfony-docs/commit/f6034cda298617ef821baca00b9da54566fbde36

### CODE REVIEW

**Comment by OskarStark** [comment]
Rebase unlocked ✅

File: notifier.rst [hunk 1]
```diff
@@ -1011,7 +1002,9 @@ is dispatched. Listeners receive a
 .. _`OneSignal`: https://github.com/symfony/symfony/blob/{version}/src/Symfony/Component/Notifier/Bridge/OneSignal/README.md
 .. _`OrangeSms`: https://github.com/symfony/symfony/blob/{version}/src/Symfony/Component/Notifier/Bridge/OrangeSms/README.md
 .. _`OvhCloud`: https://github.com/symfony/symfony/blob/{version}/src/Symfony/Component/Notifier/Bridge/OvhCloud/README.md
+.. _`PageDuty`: https://github.com/symfony/symfony/blob/{version}/src/Symfony/Component/Notifier/Bridge/PageDuty/README.md
```
Comment (OskarStark) [inline notifier.rst]:
```suggestion
.. _`PagerDuty`: https://github.com/symfony/symfony/blob/{version}/src/Symfony/Component/Notifier/Bridge/PageDuty/README.md
```

**Comment by mikaelkael** [comment]
@OskarStark : I correctly rebased for merge :blush:

**Review by OskarStark — APPROVED** [review]

**Comment by OskarStark** [comment]
Thank you Mickael.

**Pull request merged**